Makefile

```diff
@@ -24,5 +25,10 @@ clean:
 	docker rmi nextlink/releasemaster || :
 
 test:
-	npm install
-	npm test
+	clear
+	echo Starting test *********************************************************
+	./node_modules/mocha/bin/mocha \
```

Comment:
this won't work, if you don't run 'npm install' .
I prefer to put all test command in package.json , use "make target" to trigger the "npm test"